tests/integration/piv_test.py

```diff
@@ -805,3 +863,49 @@ def test_piv_modification_limitation(mock_rabbitmq):
             assert db_trip_partial_removed.contributor_id == PIV_CONTRIBUTOR_ID
 
     assert mock_rabbitmq.call_count == 1
+
+
+def test_piv_partial_back_ok(mock_rabbitmq):
+    piv_feed = _get_stomp_20201022_23187_delayed_5min_fixture()
+    res = api_post("/piv/{}".format(PIV_CONTRIBUTOR_ID), data=ujson.dumps(piv_feed))
+
+    assert "PIV feed processed" in res.get("message")
+
+    _assert_db_stomp_20201022_23187_delayed_5min()
+    # the rabbit mq has to have been called twice
+    assert mock_rabbitmq.call_count == 1
+
+    piv_feed = _get_stomp_20201022_23187_partial_delayed_fixture()
+    retard = {"duree": 0, "dureeInterne": 3}
+    _set_event_on_stops(
+        fixture=piv_feed,
+        disruption_type="NORMAL",
+        retard_dict=retard,
+        rang_min=4,
+        rang_max=5,
```

Comment:
Not sure and curious, why not:
```suggestion
        rang_max=6,
```

Reply:
in blank fixture the maximum "rang" is 5

Reply:
Ah OK, there's a rang=0.
What "bugged" me is that there are 4 stops "normal" and you only put piv_disruption on the last 2.
It's OK to test a bit more (or weirder cases) though.

Reply:
done in 1f95031ffa55c65a375ba84172c2c8ab176065e6